src/spc_django/settings.py

```diff
@@ -148,8 +148,12 @@
 
 STATIC_URL = '/static/'
 
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "staticfiles"),
+]
+
 LOGIN_REDIRECT_URL = '/'
 LOGOUT_REDIRECT_URL = '/login/'
 
-MEDIA_ROOT = os.path.join(BASE_DIR, '')
-MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+MEDIA_URL = '/staticfiles/'
```

Comment:
You need to fix this, take a reference of how I am saving documents in JobAdvertisement, in the JAF field